cooltools/cli/coverage.py

```diff
@@ -80,15 +88,19 @@ def coverage(
 
     try:
         cis_cov, tot_cov = api.coverage.coverage(
-            clr, ignore_diags=ignore_diags, chunksize=chunksize, map=_map, store=store
+            clr, ignore_diags=ignore_diags, chunksize=chunksize, map=_map, store=store, clr_weight_name=clr_weight_name
         )
     finally:
         if nproc > 1:
             pool.close()
 
     coverage_table = clr.bins()[:][["chrom", "start", "end"]]
-    coverage_table["cis_raw_cov"] = cis_cov.astype(int)
-    coverage_table["tot_raw_cov"] = tot_cov.astype(int)
+    if clr_weight_name is None:
+        coverage_table["cis_raw_cov"] = cis_cov.astype(int)
+        coverage_table["tot_raw_cov"] = tot_cov.astype(int)
+    else:
+        coverage_table["cis_raw_cov"+str("_"+clr_weight_name)] = cis_cov.astype(float)
+        coverage_table["tot_raw_cov"+str("_"+clr_weight_name)] = tot_cov.astype(float)
```

Comment:
we've been trying to switch to python f-strings in the cooltools , IMHO they're more readable :
```python
"tot_raw_cov"+str("_"+clr_weight_name)  # VS
f"tot_raw_cov_{clr_weight_name}"
```